src/sentry/models/releaseactivity.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model
+
+
+class ReleaseActivity(Model):
+    __include_in_export__ = False
+
+    class Type:
+        unknown = 0
```

Comment:
What will we use this type for?

Reply:
Do we need a default? I think it'd be better to fail the write if a `type` isn't provided